src/Event/ErrorEvent.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Vich\UploaderBundle\Event;
+
+
+use Vich\UploaderBundle\Mapping\PropertyMapping;
+
+/**
```

Comment:
please remove this useless comment. Also, some style fixes are needed (see the actions)

Reply:
Which action by the way ? I tried to find it.

Reply:
https://github.com/dustin10/VichUploaderBundle/pull/1450/checks

Reply:
Thanks! Fixed now :) It seems to me that the PHPStan one is not related to this PR

Reply:
Not yet.

By the way, I see some English errors in your text, please check it with some language tool (something like Grammarly)

Reply:
My bad. I focused on the markdown issues.

Reply:
I also tried to improve the language as per your input :)